.github/workflows/build_wheels_windows.yml

```diff
@@ -182,17 +221,42 @@ jobs:
           else
             ${CONDA_RUN} ${ENV_SCRIPT} python setup.py bdist_wheel ${BUILD_PARAMS}
           fi
+      - name: Build the wheel (bdist_wheel) Arm64
+        if: inputs.architecture == 'arm64'
+        env:
+          PACKAGE_NAME: ${{ inputs.package-name }}
+          DOWNLOADS_DIR: c:\temp\downloads\
+          DEPENDENCIES_DIR: c:\temp\dependencies\
+          SCRIPTS_DIR: test-infra\\.github\\scripts\\winx64test
+          SRC_DIR: ${{ inputs.repository }} 
+        shell: cmd
+        run: |
+          set VS_PATH=%DEPENDENCIES_DIR%\VSBuildTools\VC\Auxiliary\Build\vcvarsall.bat
+          set GIT_BASH=%DEPENDENCIES_DIR%\git\usr\bin\bash.exe
+          cd %SCRIPTS_DIR%
+
+          if "${{ env.PACKAGE_NAME }}" == "torchaudio" (
+          call "%VS_PATH%" arm64
+          "%GIT_BASH%" -c "bash --noprofile --norc build_audio.sh"
+          ) else if "${{ env.PACKAGE_NAME }}" == "torchvision" (
+          call "%VS_PATH%" arm64
+          "%GIT_BASH%" -c "bash --noprofile --norc build_vision.sh"
+          ) else (
+          echo Invalid repository name: ${{ inputs.repository }}
+          exit /b 1
+          )
```

Comment:
Why do we have scripts for building specific repositories here?

The ideal state of these workflows is that you should be able to drop these in any repository and they should "just work" that's how these currently work so why is windows arm64 different here?

Reply:
Thanks for the clarification, I misunderstood the logic of the workflow setup. I’ll move the ARM64-specific scripts to their appropriate repos and update this accordingly.